docs/index.rst

```diff
@@ -1,17 +1,17 @@
 .. You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Scripture Burrito Documentation (v0.1 Beta)
-===========================================
+Scripture Burrito Documentation
+===============================
 
 .. image:: ../logo/burrito_logo_0_1_beta.png
 
-This is the official documentation for the evolving Scripture Burrito standard. This version is 0.1 Beta.
+This is the official documentation for the evolving Scripture Burrito standard. This version is **0.1.0 Beta**.
 
 Status
 ------
 
-This standard is work in progress. Things may well change significantly before v1.0. At this point the proposal includes
+This standard is work in progress. Things may well change significantly before v1.0. At this point the proposal includes:
```

Comment:
v1.0.0, if we're being semantically pedantic?

Reply:
Fixed.